### PR TITLE
Fix some spelling mistakes in the main README and make the verbage a little more consistent

### PR DESCRIPTION
**Pull request by Multi-Volt**
This project is super cool! I noticed a few issues in the read-me so I went ahead and took a pass at reviewing it and fixed some mistakes I found in the language. Hope this helps!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,13 +1,13 @@
 SM64 Randomized Asset Generator
 ===============================
 
-Generates non-copyrighted randomized assets so sm64 and sm64-port can be used
```
Comment (Erotemic) [inline README.rst:4]: This original line is correct. I wanted to reference the names of the repo. We can put them in backticks to codify them if you think that looks better.

**Comment by Erotemic** [comment]
Thanks, the first line I'd like to tweak, but everything else looks great!

**Review by Erotemic — CHANGES_REQUESTED** [review]
Just want to change this first line, everything else looks good.

**Comment by Multi-Volt** [comment]
> Just want to change this first line, everything else looks good.

I added backticks with embedded hyperlinks to the repositories already present in the repository. I think this could make things easier to digest/access. Let me know your thoughts!

**Comment by Erotemic** [comment]
I agree these are all improvements. Thanks!

**Pull request merged**